tests/integration/kube_basic/expected_output/kube_configmap-001.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+data:
+  foo: bar
+  some.file: 'test1
+
+    test2
+
+    test3
+
+    '
```

Comment:
This looks wrong, but pasting this into `yamllint.com` results in something that looks like the input above